hq/test/aws/support/TrustedAdvisorSGOpenPortsTest.scala

```diff
@@ -65,4 +68,57 @@ class TrustedAdvisorSGOpenPortsTest extends FreeSpec with Matchers with AttemptV
       }
     }
   }
+
+  "sorting" - {
+
+    "security flags by port priority" in {
+      val portPriorityMap = Map("27017" -> 0, "27018" -> 0, "27019" -> 0, "6379" -> 0, "6380" -> 0,  "9000" -> 1, "7077" -> 1, "4040" -> 1, "8890" -> 1, "4041" -> 1, "4042" -> 1, "8080" -> 1, "22" -> 9)
+      val sgs: Gen[SGOpenPortsDetail] = for {
+        status <- Gen.oneOf("Ok", "Warning", "Error")
+        name <- Gen.alphaStr
+        id <- Gen.alphaStr
+        vpcId <- Gen.alphaStr
+        port <- Gen.oneOf(9000, 8080, 27017, 27018, 27019, 6379, 6380, 22, 4040, 8890, 4042, 4041 )
```

Comment:
Probably OK to refer to the hard-coded port list here to reduce the burden of maintaining this test.